### PR TITLE
Stellar: Inflation destination RPC and CLI

### PR DESCRIPTION
**Pull request by zapu**

### CODE REVIEW

File: go/client/cmd_wallet_set_inflation.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+		Name:         "set-inflation",
+		Usage:        "Set inflation destination address for an account",
+		ArgumentHelp: "<account id> <destination address or label>",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(cmd, "set-inflation", c)
+		},
+		Description: `Read more about Stellar inflation payouts:
+   https://www.stellar.org/developers/guides/concepts/inflation.html
+
+   Destination can be either a user chosen address (any Stellar account ID),
```
Comment (patrickxb) [inline go/client/cmd_wallet_set_inflation.go]: user-chosen

File: go/client/cmd_wallet_set_inflation.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+   or one of the following labels:
+
+   * "lumenaut" - Sets account's inflation destination to Lumenaut Pool. Read more
+     on https://pool.lumenaut.net/
+
+   * "self" - Sets inflation destination to the source account ID.
+
+   Inflation destination cannot be removed or cleared from the account right
+   now, but if you want to stop contributing to currently set destination, use
```
Comment (patrickxb) [inline go/client/cmd_wallet_set_inflation.go]: to the currently
use the "self" option.

File: go/stellar/remote/remote.go [hunk 3]
```diff
@@ -1260,3 +1260,38 @@ func ServerTimeboundsRecommendation(ctx context.Context, g *libkb.GlobalContext)
...
+	_, err = g.API.Post(apiArg)
+	return err
+}
+
+type getInflationRes struct {
+	libkb.AppStatusEmbed
+	DestinationAccountID *stellar1.AccountID `json:"destination_account_id"`
+}
+
+func GetInflationDestination(ctx context.Context, g *libkb.GlobalContext, accountID stellar1.AccountID) (res *stellar1.AccountID, err error) {
```
Comment (patrickxb) [inline go/stellar/remote/remote.go]: will this also be in stellar1.AccountDetails?

Comment (zapu) [reply go/stellar/remote/remote.go]: there is no reason not to put it there, right?

Comment (patrickxb) [reply go/stellar/remote/remote.go]: If it's there, then you can do something like `walletState.Details(ctx, accountID).InflationDestination`

**Pull request merged**